### ISSUE TITLE
Steam cloud not syncing save data, can't upload screenshots

### ISSUE THREAD

**Issue by langurmonkey** [ValveSoftware/steam-for-linux#6334]
#### Your system information

* Steam client version (build number or date): June 17, 2019 at 23:31:27, API v018
* Distribution (e.g. Ubuntu): Arch Linux
* Opted into Steam client beta?: [No]
* Have you checked for system updates?: [Yes]

#### Please describe your issue in as much detail as possible:
Since a few days ago I can't upload screenshots nor does steam cloud sync the game save data.

#### Steps for reproducing this issue:

1. Open Steam client
2. Try uploading screenshot
3. Get fail message (see attached screenshot)

#### Screenshot of error
![screenshots_error](https://user-images.githubusercontent.com/1049893/59965448-7403d600-950e-11e9-86b1-68d14873db35.jpg)

#### Log
Here is an extract of `~/.local/share/Steam/log/cloud_log.txt` with the relevant upload log.
```
[2019-06-22 16:50:04] Starting screenshot upload job.

[2019-06-22 16:50:04] [GameID 292030] Processing screenshot #1/1 - 292030/screenshots/20190621234014_1.jpg
[2019-06-22 16:50:04] [AppID 760] HTTP upload for file '292030/screenshots/20190621234014_1.jpg' beginning . . .
[2019-06-22 16:50:30] [AppID 760] HTTP upload for file '292030/screenshots/20190621234014_1.jpg' (offset=0, length=427246) to host 'steamclouduseast.blob.core.windows.net', path /cloud/53/61/CA/05/760/02B_3_1B330FD3_6C742_1.dat - timed out.
[2019-06-22 16:50:31] [GameID 292030] Uploading 292030/screenshots/20190621234014_1.jpg finished with result Failure
[2019-06-22 16:50:31] BUploadScreenshot - BYieldingUploadSingleFile failed for 292030/screenshots/20190621234014_1.jpg
[2019-06-22 16:50:31] Screenshot upload job complete - uploaded 0 of 1
```

**Comment by kisak-valve** [comment]
Hello @langurmonkey, what kernel version are you using? Is the output of `nstat -az | grep -i wqueue` greater than zero and increasing when steam tries to sync?

**Comment by langurmonkey** [comment]
Hi there. Kernel is `5.1.12-arch1-1-ARCH`. The command
```
watch -n 1 "netstat -az | grep -i queue"
```
does indeed increase from 0 till 7. When the upload fails it looks like this:
```
TcpExtTCPWqueueTooBig           7                  0.0
```
Is this related to #6326?

**Comment by kisak-valve** [comment]
It reads like the same issue to me, should be fixed with the 5.1.14 kernel release.

**Comment by langurmonkey** [comment]
Confirmed, kernel `5.1.14-arch1-1-ARCH` fixes the issue.
